javascript-conditionals/main.js

```diff
@@ -0,0 +1,465 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro,
+            recommendMovie
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+console.log(isUnderFive(10));
+
+// iseven
+
+function isEven(number) {
+  if (number % 2 !== 0) {
+    return false;
+  } else {
+    return true;
+  }
+}
+console.log(isEven(10));
+console.log(isEven(4));
+console.log(isEven(5));
+
+// startingStr
+
+function startsWithJ(string) {
+  if (string[0] === 'J' || string[0] === 'j') {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+console.log(startsWithJ('JavaScript'));
+console.log(startsWithJ('savaScript'));
+
+// drinking if statment
+
+function isOldEnoughToDrink(person) {
+  if (person.age < 21 || person.age <= 18) {
+    return false;
+  } else {
+    return true;
+  }
+}
+
+console.log(isOldEnoughToDrink({ name: 'Bartholomew JoJo Simpson', age: 10 }));
+console.log(isOldEnoughToDrink({ name: 'Bartholomew JoJo Simpson', age: 100 }));
+console.log(
+  isOldEnoughToDrink({ name: 'Bartholomew JoJo Simpson', age: 103450 })
+);
+
+// drinving if statment
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+console.log(isOldEnoughToDrive({ name: 'Jack', age: 22 }));
+console.log(isOldEnoughToDrive({ name: 'idk', age: 2 }));
+
+// drinking and driving yayyy
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person) {
+    return false;
+  } else {
+    return 'YOU DONT EVEN EXIST';
+  }
+}
+
+console.log(isOldEnoughToDrinkAndDrive({ name: 'idk', age: 2 }));
+console.log(isOldEnoughToDrinkAndDrive());
+
+// ph levels
+
+function categorizeAcidity(pH) {
+  if (pH === 7) {
+    return 'Nuetral';
+  } else if (pH <= 7 && pH > 0) {
```

Comment:
This should be `< 7` & `>= 0`. `7` is considered neutral, so the number shouldn't be equal to it, and `0` is still considered acid so it needs to be greater than or equal to.